glide-core/build.rs

```diff
@@ -13,10 +13,11 @@ fn build_protobuf() {
         .input("src/protobuf/response.proto")
         .input("src/protobuf/connection_request.proto")
         .customize(customization_options)
+        .out_dir("src/generated")
```

Comment:
It is not mandatory, but it puts generated files to a place where they could be found